packages/grid/src/vaadin-grid-styles.js

```diff
@@ -32,6 +32,13 @@ registerStyles(
       pointer-events: none;
     }
 
+    :host([disable-text-selection]) {
+      -moz-user-select: none;
+      -webkit-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
+    }
```

Comment:
In Firefox, text selection is not prevented when dragging.

Reply:
As neither user-select nor -moz-user-select work as expected in Firefox, added some JavaScript code to clear text selection.

Reply:
Fixed